torch/_ops.py

```diff
@@ -46,6 +47,23 @@ def dl_open_guard():
         sys.setdlopenflags(old_flags)
 
 
+class Kernel:
```

Comment:
put this somewhere else. Maybe in torch/_dispatch or torch/_library. Otherwise I think this gets exposed as torch.ops.Kernel (things in torch._ops behave a little weirdly)